{{cookiecutter.project_slug}}/.github/workflows/pytest.yml

```diff
@@ -1,10 +1,21 @@
-name: Unit Tests 
+name: Unit Tests
 on: [push]
 jobs:
   pytest:
     runs-on: ubuntu-latest
     env:
```

Comment:
@oudeismetis can you help me with this? The config matches all the projects I swiched to psql but the error here seems to indicate maybe the db is not created? Is there maybe another place this should go ?

Reply:
for future readers...
This issue here was that the bootstrapper never runs the Github Actions for the generated project. It has it's own action for running Pytest. So a change like this has to happen in 2 places.